msal/authority.py

```diff
@@ -91,8 +99,9 @@ def __init__(self, authority_url, http_client, validate_authority=True):
             tenant_discovery_endpoint = payload['tenant_discovery_endpoint']
         else:
             tenant_discovery_endpoint = (
-                'https://{}{}{}/.well-known/openid-configuration'.format(
+                'https://{}:{}{}{}/.well-known/openid-configuration'.format(
```

Comment:
need / after port?

Reply:
It did not have `/` in previous implementation. Looks like the `path` already contains a leading `/`.

Reply:
correct, path already starts with /.